spring-cloud-netflix-eureka-server/src/main/java/org/springframework/cloud/netflix/eureka/server/EurekaServerConfigBean.java

```diff
@@ -190,6 +190,14 @@
 
 	private int minAvailableInstancesForPeerReplication = -1;
 
+	private int initialCapacityOfResponseCache = 1000;
```

Comment:
Where are these defaults coming from?

Reply:
Because the default value of the eureka server guava cache in the existing version is 1000.

see https://github.com/Netflix/eureka/pull/1096